modules/tinymce/src/core/test/ts/browser/focus/HighlightOnFocusTest.ts

```diff
@@ -14,11 +15,16 @@ describe('browser.tinymce.core.focus.HighlightOnFocus', () => {
     await Waiter.pWait(200);
   };
 
+  const pAssertIsNotHighlighted = (editor: Editor) =>
+    Waiter.pTryUntil('Waiting for focus to be shifted', () => assertIsNotHighlighted(editor));
+
   context('with highlight_on_focus: true', () => {
     const hook = TinyHooks.bddSetup<Editor>({
       base_url: '/project/tinymce/js/tinymce',
+      plugins: 'searchreplace',
+      toolbar: 'searchreplace forecolor',
       highlight_on_focus: true
-    });
+    }, [ SearchReplacePlugin ]);
```

Comment:
Why is the `searchreplace` plugin in this test?

Reply:
Yeah, I added the plugin in the test for testing that it doesn't highlight the content area when I opened an inline dialog, and then close it. Would a custom dialog spec be more appropriate in this case?

Reply:
Yea that would work best. Perhaps it could be added to the core test utils to prevent duplication in the future.